SKILL.tsv

```diff
@@ -1267,7 +1267,7 @@ SKILL_20150317_001266	{memo X}Buff for adding attack speed
 SKILL_20150317_001267	{memo X}Make ground skill of enemies be usable by allies.
 SKILL_20150317_001268	{memo X}test
 SKILL_20150317_001269	$Multi Shot
-SKILL_20150317_001270	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Quickly launch multiple arrows at a specified location.
+SKILL_20150317_001270	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Quickly launch multiple arrows at an enemy.
```

Comment:
Like I told you in Discord, you must not rely on videos. That is a bad way of handling translation. The description tells you more about the functionality than a video. 

That video is not updated. If you cast Multi Shot, a magic circle appears that you can move to a specified location. So yes, there is a specified location.